test/emqx_acl_cache_SUITE.erl

```diff
@@ -80,6 +97,28 @@ t_reload_aclfile_and_cleanall(_Config) ->
     ?assert(length(gen_server:call(ClientPid, list_acl_cache)) > 0),
     emqtt:stop(Client).
 
+
+%% asserts that we check ACL for the LWT topic before publishing the
+%% LWT.
+t_last_will_testament_message_check_acl(_Config) ->
+    ClientID = <<"lwt_acl">>,
+    {ok, C} = emqtt:start_link([
+        {clientid, ClientID},
+        {will_topic, <<"$SYS/lwt">>},
+        {will_payload, <<"should not be published">>}
+    ]),
+    {ok, _} = emqtt:connect(C),
+    ok = emqx:subscribe(<<"$SYS/lwt">>),
+    unlink(C),
+    exit(C, kill),
+    receive
+        {deliver, <<"$SYS/lwt">>, #message{payload = <<"should not be published">>}} ->
+            error(lwt_should_not_be_published_to_forbidden_topic)
```

Comment:
use `ct:fail` instead?

Reply:
this is nit

Reply:
I'll try to use `ct:fail` on the future PRs, thanks!  :beers: 

This time, I'll merge it as is to avoid running the CI for that change, if you don't mind.  :smiley_cat: